.github/workflows/noora.yml

```diff
@@ -30,7 +36,7 @@ jobs:
         with:
           experimental: true
       - name: Run
-        run: mise run build
+        run: mise run build-cli
```

Comment:
What about the following structure?

```
.mise/
  tasks/
    cli/
      build.sh
      lint.sh
      test.sh
    web/
      ..
```

Which you can invoke with `mise run cli:build`. Alternatively, we can place the action first: `mise run build:cli`. I think I like this one better :)